.chloggen/fix_compressor-kinesis-exporter-thread-safe.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: 'bug_fix'
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: awskinesisexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: the compressor was crashing under high load due it not being thread safe.
+
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
+issues: [32589]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: Instead of using a mutex in the compressor Do I ended up creating a new compressor on each batch generation because it generated more performant execution in my tests with heavy load, specially if the payloads were bigger
```

Comment:
This subtext reads more as a first person introspection as opposed to more informative details.

Could I ask you to keep it focused on the improvement?